api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](*) and [devices](*) (and more?) who share access to resources in Office365 or in your app. All operations in Microsoft Graph on groups require administrator consent. Using Microsoft Graph, you can perform the following operations and more:

```

Comment:
So I'm not quite sure about this opening statement. AFAIK Office 365 groups (all flavors) can only have users as members, Security groups can have users and groups, Outlook distribution groups can have users, groups and contacts. Not sure about devices for any of these and a solid emphasis on "AFAIK"  :). Also, O365 groups are all about collaboration, while security groups are about controlling access to resources in a directory or app(?).